api/client/client.go

```diff
@@ -803,7 +803,12 @@ func (c *Client) GetAccessRequests(ctx context.Context, filter types.AccessReque
 			break
 		}
 		if err != nil {
-			return nil, trail.FromGRPC(err)
+			err := trail.FromGRPC(err)
+			if trace.IsNotImplemented(err) {
```

Comment:
Do we actually need the not implemented error in both places, or can we remove the original one?

Reply:
@codingllama It does sometimes unfortunately. gRPC doesn't specify when this happens either and I cannot find solid API docs or guarantees. I'd lean on keeping this in so we don't get smacked when we update our protobuf compiler or something.

Reply:
In your tests, which one do we hit? Does it change if you use an actual gRPC server with the streaming RPC removed?

Let's add a comment saying we are keeping both checks for safety, but no objections on my part.

Reply:
@xacrimon let's find the specific place(s) where we see the not implemented error and not guess as to where it might be. The most realistic test to run is testing with plugins - @r0mant can walk through what steps to run if you need clarification.

Reply:
Removed

Reply:
Yeah, I realized that if a dep does ever change like this, this test will fail anyway. Which makes it safe to remove this.